src/App.jsx

```diff
@@ -14,70 +15,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {value === '' ? (
+        <h1
+          className="title is-flex is-align-items-center"
+        >
+          No goods selected
+        </h1>
+      ) : (
+        <div>
+          <h1 className="title is-flex is-align-items-center">{`${value} is selected`}</h1>
+          <button
+            onClick={() => {
+              setValue('');
+            }}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        </div>
+      )}
+      <div>
+        <table className="table">
+          <tbody>
+            {goods.map(word => (
```

Comment:
good is a better name that word, because it's element of goods 
```suggestion
            {goods.map(good => (
```